node/heartbeat/sender.go

```diff
@@ -66,7 +72,7 @@ func (s *Sender) SendHeartbeat() error {
 
 	hb := &Heartbeat{
 		Payload:         []byte(fmt.Sprintf("%v", time.Now())),
-		ShardID:         s.shardCoordinator.SelfId(),
+		ReceivedShardID: s.shardCoordinator.SelfId(),
```

Comment:
Since this is awkward, might leave the Heartbeat's field plain 'ShardId' and have only ReceivedShardId and ComputedShardId in heartbeat message info

Reply:
+1

Reply:
done